opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -681,6 +686,80 @@ export const meEditField = async (context, user, userId, inputs, password = null
   return userEditField(context, user, userId, inputs);
 };
 
+export const deleteAllWorkspaceForUser = async (userId) => {
+  return await elRawDeleteByQuery({
```

Comment:
I'm wondering if having these `elRawDeleteByQuery` in a domain class is thre right place or if it should be kepts in engine.js ?

Reply:
I had the same interrogation, but I saw that @richard-julien did similar stuff for "works" in `domain/work.js l150` very recently so..

Reply:
For me there is no issue to use engine.js in domain files. Should be limited because lot of stuff is handled by middleware.js but when you need to do low level api, it seems ok for me. Engine.js is not the place to have specific function linked to specific entity behavior so i dont think its a good idea to move the function here. I dont really see a good alternative but if you have some idea, feel free to propose :)